dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -16,19 +17,38 @@
  *
  * <pre>
  *   <em>new constructor(arg1, arg2, ...)</em>
+ *   <em>enclosingExpression.new constructor(arg1, arg2, ...)</em>
  * </pre>
  */
 public class ObjectCreationNode extends Node {
 
+    /** The tree for the object creation. */
     protected final NewClassTree tree;
+
+    /** The enclosing type receiver of the object creation, which can be nullable. */
```

Comment:
What is the `enclosing type receiver`?
Instead of just saying that it is nullable (which one already sees from the annotation), say when it is non-null and when it can be null.

Reply:
Thanks!